examples/events.rs

```diff
@@ -0,0 +1,134 @@
+//! Adapted from https://github.com/bevyengine/bevy/blob/v0.9.1/examples/2d/sprite_sheet.rs
+//! and https://github.com/bevyengine/bevy/blob/v0.9.1/examples/animation/animated_fox.rs
+//! Renders an animated sprite by loading all animation frames from multiple sprites
+//! and changing the displayed image periodically.
+
+#[path = "helpers/animation_controller.rs"]
+mod animation_helper;
+
+use animation_helper::keyboard_animation_control_helper;
+use bevy::prelude::*;
+use bevy_trickfilm::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(ImagePlugin::default_nearest())) // prevents blurry sprites
+        .add_plugins(Animation2DPlugin)
+        .add_event::<SampleEvent>()
+        .add_animation_event::<SampleEvent>()
+        .register_type::<SampleEvent>()
```

Comment:
Is it possible to have just a call to `add_animation_event` and register the type from that call? And also the `add_event` call? It seems quite redundant to have all 3, but I guess it wouldn't be that easy to implement.

Reply:
Yes, it is. I did not do that yet, because I thought from a user perspective it might be weird that you don't see any add_event for your Event.
But I see that bevy_eventlistener e.g. also does it they way you suggested, so I will combine those calls.